app/assets/stylesheets/_tables.scss

```diff
@@ -0,0 +1,36 @@
+[aria-sort] > * {
+  position: relative;
+
+  &:after {
+    font-size: em(10, 19);
```

Comment:
Updated to use pixel sizing - didn't use mixin as it was clearly to read it with plain pixel values .